spec/requests/api/v1/documents_spec.rb

```diff
@@ -2,64 +2,68 @@
 require 'uri'
 
 describe API::V1::Documents do
-  let(:id) { "some really!weird@id.name" }
-  let(:valid_params) do
-    {
-      "document_id" => id,
-      "title" => "my title",
-      "path" => "http://www.gov.gov/goo.html",
-      "description" => "my desc",
-      "promote" => true,
-      "language" => 'hy',
-      "content" => "my content",
-      "tags" => "Foo, Bar blat",
-    }
+  let(:id) { 'some really!weird@id.name' }
+
```

Comment:
We normally don't have a line break between `let` blocks. We generally follow "Better Specs" as an RSpec style guide. See http://www.betterspecs.org/#let

Reply:
Yeah, the standard is to `"group let, subject blocks and separate them from before/after blocks"` I.e.:
```
let(:foo) {}
let(:bar) {}

subject(:baz) {}

before(:quux) {}
```
https://github.com/rubocop-hq/rspec-style-guide#let-grouping

Reply:
👍 will update that.